tests/Main.hs

```diff
@@ -14,6 +14,7 @@ import           Tests.Permutations                          (specPermutations)
 import           Tests.Plonk                                 (specPlonk)
 import           Tests.Scripts.LockedByTxId                  (specLockedByTxId)
 import           Tests.UInt                                  (specUInt)
+import           Tests.ByteString (specByteString)
```

Comment:
Could you use a stylish-haskell formatter to format this line? We have configs for VS Code and NeoVim in this repository, but could support more editors if needed

Reply:
Just missed this file. Of course, I always run stylish-haskell :)